recipe/meta.yaml

```diff
@@ -1,18 +1,23 @@
 {% set name = "transformers" %}
-{% set version = "4.24.0" %}
+{% set version = "4.29.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 486f353a8e594002e48be0e2aba723d96eda839e63bfe274702a4b5eda85559b
+  sha256: ed9467661f459f1ce49461d83f18f3b36b6a37f306182dc2ba272935f3b93ebb
 
 build:
   skip: True  # [py<37]
+  # s390x is missing pyrarrow that datasets depends on
+  skip: True  # [linux and s390x]
+  # At present, there are no 3.11 compatible Pytorch version for win or ppc
+  # There is a bug requiring PyTorch 1.10 on osx-64 which is not available for 3.11 
+  skip: True  # [py>310 and (ppc64le or win or (osx and x86_64))]
```

Comment:
That will extremely narrow the possibility to build `spacy-transformers` and consequently space data models (particularly, `trf` models like this https://github.com/AnacondaRecipes/spacy-models-feedstock/blob/a354664bc150819e5d45bb51d99e5f5e26a88aa2/recipe/meta.yaml#L443) but we can skip building trf spacy data models by modifying this automate build script https://github.com/AnacondaRecipes/spacy-models-feedstock/blob/main/recipe/_update_spacy_recipe.py.
As we do not have pytorch we can go only this way